### ISSUE TITLE
Publish Helm chart on releases

### ISSUE THREAD

**Issue by yorugac** [grafana/k6-operator#277]
### Feature Description

There should be a CI workflow on k6-operator releases publishing Helm chart to the repository.

### Suggested Solution (optional)

It appears that all (?) Grafana Helm charts are published here:
https://grafana.github.io/helm-charts/

These can likely be used as code references:
- https://github.com/grafana/helm-charts/blob/main/.github/workflows/update-helm-repo.yaml
- https://github.com/grafana/agent/blob/main/.github/workflows/helm-release.yml

### Already existing or connected issues / PRs (optional)

Depends on:
- PR #98 being merged

**Comment by mladenrtl** [comment]
I have the same question. I am waiting this to be released to helm-charts repository, so I can use it as dependency in our GitOps project, like:
```
dependencies:
  - name: k6-operator
    version: 1.0.0
    repository: https://grafana.github.io/helm-charts
    alias: k6-operator
```
I tried to use this but I am getting error:
`helm dependency build failed exit status 1: Error: k6-operator chart not found in repo https://grafana.github.io/helm-charts`

**Comment by mabre69uk** [comment]
until the helm chart will not be release in the grafana repository, will not be usable...can we have an idea of when this will be completed? thanks a lot

**Comment by mladenrtl** [comment]
What I do currently, I copy the chart directory to our GitOps repo and use it like that. It works fine, but it's not the convenient way, especially if there are some changes to the chart.

**Comment by yorugac** [comment]
Hello :wave: 
Just to be clear, this issue is not closed yet. So no, our Helm is not published anywhere at the moment. It's an active WIP though. 
I'll update here when it's done :+1:

**Comment by mladenrtl** [comment]
@yorugac Thank you for your reply. Please let me know if you need any help with it, I would like to contribute

**Comment by mladenrtl** [comment]
@yorugac Thank you very much for the update. I will track this issue, and I hope it will be fixed soon.

**Comment by yorugac** [comment]
A small but happy update :slightly_smiling_face:  The epic with Github authentication to helm-charts is finally looking close to the finish line.The 1st version of Helm chart was published yesterday + bump to 0.0.11 today and can now be installed with these commands:
```
helm repo add grafana https://grafana.github.io/helm-charts
helm install k6-operator grafana/k6-operator
```

**Comment by yorugac** [comment]
Related PR: https://github.com/grafana/helm-charts/pull/2728

Regardless of what happens with this PR to helm-charts, I believe this issue can be considered done: right now, Helm release will be triggered on any change to `Chart.yaml` in main branch.

This implies that updates of Helm will require manual bumps of versions in `Chart.yaml` at the moment. It'd be good to somehow automate that in the future as well since releases in k6-operator are becoming too cumbersome will all these manual steps; but for now I consider it sufficient.
